bootstrap.php

```diff
@@ -318,6 +318,9 @@
     ->rule('Gdn_Model')
     ->setShared(true)
 
+    ->rule(Model::class)
+    ->setShared(true)
```

Comment:
```suggestion
    ->rule(Model::class)
    ->setInherit(true)
    ->setShared(true)
```

Does this actually do anything without a `setInherit`? All of our model instances are subclasses, and we never pull them out of the container as just `Model`.

Reply:
I had checked the code and I think inherit is the default:

```
// Don't add the rule if it doesn't say to inherit.
if (!isset($this->rules[$class]) || (isset($this->rules[$class]['inherit']) && !$this->rules[$class]['inherit'])) {
    continue;
}
```

Reply:
I'll confirm with a debug session too. One sec...

Reply:
Okay. I confirmed that inherit works, but there was a bug with the rule. I didn't import the namespace.